test/Tool/signbits6.opt

```diff
@@ -0,0 +1,10 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i32 = var (p) (s=31)
+%1:i32 = cttz %0
+%2:i1 = eq %1, 0:i32
+cand %2 1:i1
```

Comment:
In my understanding, here the lowest significant bit has to be 1. I can still confirm if the 32-bit number with value 0, has 32 sign bits or 31.